### PR TITLE
Silly sintouched objectives

### PR DESCRIPTION
**Pull request by ghost**
:cl: Reeeeimstupid
add: Several funny Sintouched objectives. But again, most of them weren't serious, either.
/:cl:

Sintouched is interesting. I gave it a few more silly objectives.

### CODE REVIEW

**Comment by gt33** [comment]
Ok, it seems the test has failed. I don't see the problem, though.

File: code/game/gamemodes/miniantags/sintouched/objectives.dm [hunk 1]
```diff
@@ -32,4 +32,18 @@
...
+	explanation_text = "Angels, devils, good, evil... who cares?  Just ignore any hellish threats and do your job."
+
+/datum/objective/sintouched/engine
+	explanation_text = "Go have a good conversation with the Singularity/Tesla/Supermatter crystal. Bonus points if it responds."
+	
+/datum/objective/sintouched/teamredisbetterthangreen
+	explanation_text = "Tell the AI (or a borg/pAI/drone if there is no AI) some corny technology jokes until it cries for help."
+	
+/datum/objective/sintouched/time
+	if(istype(ticker.mode, /datum/game_mode/clock_cult))
```
Comment (RemieRichards) [inline code/game/gamemodes/miniantags/sintouched/objectives.dm]: the error is here, you're trying to run code in a definition, you can't do that, so it sees ```/datum/game_mode/clock_cult``` as a new definition, which also isn't allowed as clock cult is already defined.

A simple fix would be to add ```/New()``` to the end of line 43 so it becomes ```/datum/objective/sintouched/time/New()```, and then on the line just below it (moving everything else down a line) adding ```..()```

Comment (gt33) [inline code/game/gamemodes/miniantags/sintouched/objectives.dm]: Thank you very much.

Comment (RemieRichards) [inline code/game/gamemodes/miniantags/sintouched/objectives.dm]: :)

**Comment by gt33** [comment]
gonna close/reopen so travis CI tests the right commit.

**Comment by gt33** [comment]
@RemieRichards, I'm guessing I need to add /clock_cult?

**Comment by RemieRichards** [comment]
@gt33 it's actually due to clock cult being called ```/datum/game_mode/clockwork_cult``` by the code, so just swap the istype to checking for that instead.

**Comment by gt33** [comment]
Ohk, thanks

**Comment by gt33** [comment]
Yay, it compiles.

**Comment by Magikarpet** [comment]
I think an exception to clock cult may be a bad idea. It's in the vein of using syndi-cream to meta traitors, though I may be overestimating how hard people will abuse it.

**Comment by gt33** [comment]
I think people rarely even touch the Codex Gigas anyways. Plus if you actually do follow the objective, you'll just end up getting converted most likely anyways.

**Comment by anconfuzedrock** [comment]
ever hear about the 8th deadly sin of conversation with a singulo?

**Comment by gt33** [comment]
Nope. Tell me more.

**Comment by anconfuzedrock** [comment]
it doesn't exist

**Comment by anconfuzedrock** [comment]
seriously put this shit on abductor or something this makes absolutely no sense

**Comment by gt33** [comment]
ever heard of the 19th deadly sin of sleeping? Me neither.

**Comment by anconfuzedrock** [comment]
you mean sloth?

**Comment by gt33** [comment]
Will move this to abductor, I guess. Seperate PR or same?

**Comment by anconfuzedrock** [comment]
I don't know but I'd think seperate

**Pull request closed without merging**